nxcomp/src/Children.cpp

```diff
@@ -1038,6 +1038,7 @@ int UnsetEnv(const char *name)
   }
 
   result = 0;
+  delete[] varName;
```

Comment:
hmm, isn't c++ supposed to do that implicitely for local vars?

Reply:
I am leaving it, https://stackoverflow.com/questions/441831/calling-delete-on-variable-allocated-on-the-stack